src/components/ComponentPages/TopicDetails/CurrentCampCard/index.tsx

```diff
@@ -1,37 +1,36 @@
-import { Card, Button, Descriptions } from "antd";
+import { Card, Button, Descriptions, Collapse } from "antd";
 import { useSelector } from "react-redux";
 import { RootState } from "src/store";
 
+const { Panel } = Collapse;
+
 const CurrentCampCard = () => {
   const { campRecord } = useSelector((state: RootState) => ({
     campRecord: state?.topicDetails?.currentCampRecord,
   }));
   return (
-    <Card
-      className="canCard mb-3"
-      title={<h3>Current Camp Record</h3>}
-      actions={[
-        <>
+    <Collapse accordion={true} defaultActiveKey={['1']} expandIconPosition="right" className="topicDetailsCollapse">
+      <Panel header={<h3>Current Topic Record</h3>} key="1">
+        <Descriptions column={1}>
+          <Descriptions.Item label="Camp Name">
```

Comment:
May be the associate the array somewhere in the config file and iterate it here.